cmd/clusternet-estimator/app/app.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2022 The Clusternet Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package app
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	"k8s.io/klog/v2"
+
+	"github.com/clusternet/clusternet/pkg/estimator"
+	_ "github.com/clusternet/clusternet/pkg/features"
+	"github.com/clusternet/clusternet/pkg/version"
+)
+
+var (
+	// the command name
+	cmdName = "clusternet-estimator"
+)
+
+// NewClusternetEstimatorCmd creates a *cobra.Command object with default parameters
+func NewClusternetEstimatorCmd(ctx context.Context) *cobra.Command {
```

Comment:
Add a TODO for later review.

use a new cmd or not ?

Reply:
Done